src/EnsembleKalmanProcess.jl

```diff
@@ -65,6 +65,46 @@ struct FailureHandler{P <: Process, FM <: FailureHandlingMethod}
 end
 
 
+function default_options_dict(process::P) where {P <: Process}
+    if nameof(typeof(process)) == :Inversion
```

Comment:
can also do `if isa(process, Inversion)` and similar for the remaining ifelse conditionals

Reply:
I think we had some issues with the parametric typing for this type of call. I can check but Ill take a look later.

Reply:
Seems to go through! I'm running tests to confirm!